pandas/_libs/lib.pyx

```diff
@@ -1677,6 +1676,7 @@ cpdef bint is_datetime64_array(ndarray values):
     return validator.validate(values)
 
 
+# TODO: only non-here use is in test
```

Comment:
If so, I'm okay with removing in this PR.

Reply:
Poor wording on my part.  This function is used inside lib.pyx, but its only non-cython use in in tests.  So the option is to make it `cdef` instead of `cpdef` at the cost of not testing it directly.

Reply:
Ah okay. Thanks for clarifying that.